### ISSUE TITLE
Settings page

### ISSUE THREAD

**Issue by FunctionalMetatable** [jeffalo/ocular#88]
**Is your feature request related to a problem? Please describe.**
I'd like to see 3.0 scratchblocks in ocular, but scratch uses the 2.0 Plugin.

**Describe the solution you'd like**
A settings part of ocular (maybe in the dashboard) that you can customize certain settings in ocular like enabling 3.0 scratchblocks, and beta features.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by CluckCluckChicken** [comment]
some more customization would be epic, +1

**Comment by jeffalo** [comment]
let's list some things we can have in the settings. i personally think scratchblocks style would be cool, and maybe some options like attempting to view tinypic images from the wayback machine. maybe even elements of #87

**Comment by CluckCluckChicken** [comment]
would showing signatures under posts be an option as well? i remember you rejected that for some reason.

**Comment by Time-dragon** [comment]
:+1:

**Comment by jeffalo** [comment]
> would showing signatures under posts be an option as well? i remember you rejected that for some reason.

yeah. they take up space and can be annoying.. perhaps i can try getting it to render the signature from the time the post was made. that would be cool.

**Comment by FunctionalMetatable** [comment]
> > would showing signatures under posts be an option as well? i remember you rejected that for some reason.
> 
> yeah. they take up space and can be annoying.. perhaps i can try getting it to render the signature from the time the post was made. that would be cool.

s c r a t o r y

**Comment by rirurin** [comment]
> 
> 
> > > would showing signatures under posts be an option as well? i remember you rejected that for some reason.
> > 
> > 
> > yeah. they take up space and can be annoying.. perhaps i can try getting it to render the signature from the time the post was made. that would be cool.
> 
> Scratory?
scratory would work for more recent posts, though it doesn't look like it has any API from which you could just fetch the signature history information from

**Comment by CluckCluckChicken** [comment]
> > > > would showing signatures under posts be an option as well? i remember you rejected that for some reason.
> > > 
> > > 
> > > yeah. they take up space and can be annoying.. perhaps i can try getting it to render the signature from the time the post was made. that would be cool.
> > 
> > 
> > Scratory?
> 
> scratory would work for more recent posts, though it doesn't look like it has any API from which you could just fetch the signature history information from

scratory is just a frontend for scratchdb's signature history api

**Comment by FunctionalMetatable** [comment]
> scratory is just a frontend for scratchdb's signature history api

yes, see https://github.com/FunctionalMetatable/scratory/blob/3a491409e698393b8f0686683aab37a40331e093/pages/user/_user.vue#L77

**Comment by rirurin** [comment]
> 
> 
> > scratory is just a frontend for scratchdb's signature history api
> 
> yes, see https://github.com/FunctionalMetatable/scratory/blob/3a491409e698393b8f0686683aab37a40331e093/pages/user/_user.vue#L77

oh right, I forgor that endpoint existed
